DRAFT_RELEASE_NOTES.md

```diff
@@ -24,11 +24,12 @@ The following steps will upgrade your project to 1.9. These instructions consist
 ## Automatic Upgrades
 To reduce burden of upgrading aiSSEMBLE, the Baton project is used to automate the migration of some files to the new version.  These migrations run automatically when you build your project, and are included by default when you update the `build-parent` version in your root POM.  Below is a description of all of the Baton migrations that are included with this version of aiSSEMBLE.
 
-| Migration Name                                             | Description                                                                                                                                                                |
-|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| upgrade-tiltfile-aissemble-version-migration               | Updates the aiSSEMBLE version within your project's Tiltfile                                                                                                               |
-| upgrade-v2-chart-files-aissemble-version-migration         | Updates the helm chart dependencies within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE |
-| upgrade-v1-chart-files-aissemble-version-migration         | Updates the docker image tags within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE       |
+| Migration Name                                     | Description                                                                                                                                                                |
+|----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| upgrade-tiltfile-aissemble-version-migration       | Updates the aiSSEMBLE version within your project's Tiltfile                                                                                                               |
+| upgrade-v2-chart-files-aissemble-version-migration | Updates the helm chart dependencies within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE |
+| upgrade-v1-chart-files-aissemble-version-migration | Updates the docker image tags within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE       |
+| ml-flow-dockerfile-migration                       | Updates the docker image used for MLFlow containers within your project's deployment resources (<YOUR_PROJECT>-mlflow-docker/src/main/resources/docker) to use the latest version of the aiSSEMBLE   |
```

Comment:
A: I think we should more explicitly say what we're doing. It's not just the latest version of aissemble.  Maybe something like _"to use the bitnami/mlflow image as a base instead of the deprecated boozallen/aissemble-mlflow image"_

Reply:
Just updated, how does that look now?